src/js/timeline.js

```diff
@@ -1,90 +1,103 @@
-function getTL(){
-  document.getElementById("TL").innerHTML = "";
-  const xmlHttp = new XMLHttpRequest();
-  xmlHttp.onreadystatechange = () => {
-    if(xmlHttp.readyState === XMLHttpRequest.DONE){
-      if(xmlHttp.status === 200){
-        const timelineList = JSON.parse(xmlHttp.responseText);
-        for(let i = 0; i < timelineList.length; i++) {
-
-          const user = timelineList[i].twitterUser;
-
-          const div = document.createElement("div");
-          div.setAttribute("class", "tweet")
-
-          const userDiv = document.createElement("div");
-          userDiv.setAttribute("class", "userInfo");
-
-          const nameDiv = document.createElement("div");
-          nameDiv.setAttribute("class", "name");
-
-          const handleDiv = document.createElement("div");
-          handleDiv.setAttribute("class", "handle");
-
-          const imgDiv = document.createElement("div");
-          imgDiv.setAttribute("class", "img");
-
-          const msgDiv = document.createElement("div");
-          msgDiv.setAttribute("class", "msg");
-
-          const dateDiv = document.createElement("div");
-          dateDiv.setAttribute("class", "date");
-
-          const contentDiv = document.createElement("div");
-          contentDiv.setAttribute("class", "content");
-
-          const link = document.createElement("a");
-          link.setAttribute("class", "link");
-          link.setAttribute("target", "_blank");
-          link.setAttribute("href", `https://twitter.com/${user.twitterHandle}/status/${timelineList[i].id}`);
-
-          const img = document.createElement("img");
-          img.setAttribute("class", "img-circle");
-          img.setAttribute("src", user.profileImageUrl);
-
-          const name = document.createTextNode(user.name);
-          const handle = document.createTextNode(`@${user.twitterHandle}`);
-
-
-          const date = new Date(timelineList[i].createdAt);
-          const formattedDate = document.createTextNode(date.toLocaleDateString("en-US", {month:"short", day:"2-digit"}));
-
-          const message = document.createTextNode(timelineList[i].message);
-
-          imgDiv.appendChild(img);
-          nameDiv.appendChild(name);
-          handleDiv.appendChild(handle);
-
-          userDiv.appendChild(imgDiv);
-          userDiv.appendChild(nameDiv);
-          userDiv.appendChild(handleDiv);
-
-          link.appendChild(message);
-
-          dateDiv.appendChild(formattedDate);
+import React from "react";
+import ReactDOM from "react-dom";
+
+function UpdateTimelineButton(props) {
+  return React.createElement('button',
+    {
+      type: 'button',
+      onClick: props.onClick,
+    },
+    "Refresh Timeline");
+}
 
-          msgDiv.appendChild(link);
+class Avatar extends React.Component {
+  render() {
+    let user = this.props;
```

Comment:
you should access the user in this.props.user and not just this.props.  This makes it more future proof.